sympy/parsing/mathematica.py

```diff
@@ -1,133 +1,433 @@
 from __future__ import print_function, division
 
+from collections import defaultdict
+from itertools import product
 import re
 from sympy import sympify
 
 
-def mathematica(s):
-    return sympify(parse(s))
+class mathematica(object):
+    '''This class converts a string of a basic Mathematica expretion to
+    SymPy style. Output is a SymPy object.'''
+
+    # use this class like a function
+    def __new__(cls, s, additional_translation={}):
```

Comment:
Should be additional_translations (plural)

Reply:
I understand. If you or someone have a better idea about the name of `additional_translations`, please change it.